code/API_definitions/EdgeCloud_LcM.yaml

```diff
@@ -0,0 +1,712 @@
+    * __deleteApp__ - Removes an application from an OP, if there is a running instance of the given application, the request cannot be done.
+    * __getApp__ - Retrieves the information of a given application.
+ 
+    __Application Instance Management__
+    * __appInstantiation__  Request the OP to instatiate an instance of an application in a given Edge Cloud Node, if this parameter is not set, the OP will instantiate the applications in all the Edge Cloud Nodes available.
+    * __getAppInstance__  Retrieves the list with information of the instances related to a given application.
+    * __deleteAppInstance__ - Removes a given application instance from an Edge Cloud Node.
+
+    __Edge Cloud information__
+    *  __getEdgeCloudNodes__ List of the operator’s Edge Cloud Nodes and their status, ordering the results by location and filtering by status (active/inactive/unknown)
+    # Further info and support
+    (FAQs will be added in a later version of the documentation)
+ 
+  termsOfService: http://swagger.io/terms/
+  contact:
+    email: sp-edc@lists.camaraproject.org
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+  version: 0.2.0
+externalDocs:
+  description: Product documentation at Camara
+  url: https://github.com/camaraproject/
+ 
+servers:
+  - url: "{apiRoot}/{basePath}"
+    variables:
+      apiRoot:
+        default: http://localhost:9091
+        description: API root
+      basePath:
+        default: edge-cloud/v1
+        description: Base path for the Edge Cloud API
+ 
+security:
+  - oAuth2ClientCredentials: 
+    - nbi-mgmt
+ 
+tags:
+  - name: Application
+    description: Application Management
+  - name: Application Instance 
+    description: Application Instance Management
+  - name: Edge Cloud
+    description: Edge Cloud Nodes Available
+ 
+paths:
+  /appInstance:
+    post: 
+      tags:
+        - Application Instance
+      summary: Instantiation of an Application
+      description: Ask the OP to instantiate an application to one or several Edge Cloud Nodes with an Application as an input and an Application Instance as the output. 
+      operationId: appInstantiation
+      requestBody:
+        description: Details regarding where the application should be instantiated
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/AppInstantiation'
+        required: true      
+      responses:
+        '202':
+          description: Application instantiation accepted
+          content: 
+            application/json:
+              schema:
+                type: object
+                properties:
+                  appId:
+                    $ref: '#/components/schemas/AppId'
+                  aplicationInstace:
+                    type: array
```

Comment:
Can we describe if more than one "InstantiatedApp" object can be expected in under some conditions as aplicationInstace is defined as an array? And in that case how should information contained in InstantiatedApp objects can be used?

Reply:
It is an array because depending whether there is a filter in the request, the app could be instantiated in multiple edge nodes (eg. Region or all the nodes if no filter is parsed)